### ISSUE TITLE
Netezos related warnings

### ISSUE THREAD

**Issue by m-kus** [trilitech/tezos-unity-sdk#4]
```
Assets\TezosUnitySDK\Scripts\Helpers\NetezosExtensions.cs(21,37): warning CS0436: The type 'ContractScript' in 'C:\Users\mathi\My project (2)\Assets/TezosUnitySDK/Scripts/Netezos/Contracts/ContractScript.cs' conflicts with the imported type 'ContractScript' in 'Netezos, Version=2.6.2.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in 'C:\Users\mathi\My project (2)\Assets/TezosUnitySDK/Scripts/Netezos/Contracts/ContractScript.cs'.
```

**Comment by SamuelAsherRivello** [comment]
Of tangential relation, I see many (>999) warnings when I first open the Unity Project of the repo. (see image)

I click 'clear' in the console and they disappear, and I do not see them again on project recompilations.

I see many (>999) warnings when I build the project. I'm unsure if they are the same warnings mentioned above or not. Did not investigate.

![samr-tezos-advice-1-feedback-on-tezos-unity-sdk-google-docs-0](https://user-images.githubusercontent.com/63511769/222192652-65059da6-4dc6-4d87-87e8-56b620afdbae.png)

**Comment by k-karuna** [comment]
Fixed in #25
